test/libyul/EwasmTranslationTest.cpp

```diff
@@ -99,11 +99,10 @@ string EwasmTranslationTest::interpret()
 	InterpreterState state;
 	state.maxTraceSize = 10000;
 	state.maxSteps = 100000;
-	WasmDialect dialect;
-	Interpreter interpreter(state, dialect);
 	try
 	{
-		interpreter(*m_object->code);
+		WasmDialect dialect;
+		Interpreter::run(state, dialect, *m_object->code);
```

Comment:
Would something like this work?
```suggestion
		Interpreter::run(state, WasmDialect{}, *m_object->code);
```